fontend/app/products/[productId]/page.tsx

```
@@ -39,6 +40,91 @@ interface ApiResponse {
   status: number;
   message: string;
   data: ProductDetail;
+=======
+import { CldImage } from 'next-cloudinary'; 
```

Biome (1.9.4)

[error] Shouldn't redeclare 'CldImage'. Consider to delete it or rename it.

'CldImage' is defined here:


(lint/suspicious/noRedeclare)